settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
 
 INSTALLED_APPS = [
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
     "db",
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The `AUTH_USER_MODEL` setting is correctly configured to use the custom `User` model from the `db` app. This ensures that the custom user model is used throughout the project.